Apps/Playground/iOS/ViewController.swift

```diff
@@ -1,6 +1,71 @@
 import UIKit
 import MetalKit
 
+import UIKit
+import UIKit.UIGestureRecognizerSubclass
+
+/**
+  * A very simple gesture recognizer.  All that it does is to emulate the functionality found in other platforms
+  */
+class UIBabylonGestureRecognizer: UIGestureRecognizer {
+    // Callback for touch down events
+    public var onTouchDown: (Int32, Int32, Int32)->Void = {(pointerId: Int32, inX: Int32, inY: Int32) -> Void in
+        // Do nothing
+    }
+    // Callback for touch movement events
+    public var onTouchMove: (Int32, Int32, Int32)->Void = {(pointerId: Int32, inX: Int32, inY: Int32) -> Void in
+        // Do nothing
+    }
+    // Callback for touch up events
+    public var onTouchUp: (Int32, Int32, Int32)->Void = {(pointerId: Int32, inX: Int32, inY: Int32) -> Void in
+        // Do nothing
```

Comment:
Probably makes more sense for these to be private `let` variables and initialized by the constructor (`init`)

Reply:
Lemme see if I can get this changed over to initialize via the constructor